src/PIL/Image.py

```diff
@@ -651,14 +651,20 @@ def _repr_png_(self):
 
         :returns: PNG version of the image as bytes
         """
-        return self._repr_image("PNG", compress_level=1)
+        try:
+            return self._repr_image("PNG", compress_level=1)
+        except Exception:
+            return None
 
     def _repr_jpeg_(self):
         """iPython display hook support for JPEG format.
 
         :returns: JPEG version of the image as bytes
         """
-        return self._repr_image("JPEG")
+        try:
+            self._repr_image("JPEG")
```

Comment:
```suggestion
            return self._repr_image("JPEG")
```

Reply:
Heh, I've been writing too much rust lately and keep forgetting the returns in python :)